public/types/index.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+export interface DocumentsIndex {
+  'docs.count': string;
+  'docs.deleted': string;
+  health: 'green' | 'yellow';
+  index: string;
+  pri: string;
+  'pri.store.size': string;
+  rep: string;
+  status: 'open' | 'close';
+  'store.size': string;
+  uuid: string;
+}
+
+export interface Document {
+  _index: string;
+  _id: string;
+  _score: number;
+  _source: {
+    id: string;
+    name: string;
+    title: string;
+    short_description: string;
+    img_500x500: string;
+    date_released: string;
+    supplier: string;
+    price: number;
```

Comment:
These fields are just here for the mock, right? The documents returned from the search requests may have any fields (and the two search requests are not guaranteed to return the same sets of fields).

Reply:
Just know that... thanks for pointing out.
I will remove these in next commit.